deepchem/feat/graph_data.py

```diff
@@ -77,75 +73,77 @@ def __init__(
     self.edge_index = edge_index
     self.edge_features = edge_features
     self.graph_features = graph_features
-    self.targets = targets
     self.num_nodes, self.num_node_features = self.node_features.shape
     self.num_edges = edge_index.shape[1]
     if self.node_features is not None:
       self.num_edge_features = self.edge_features.shape[1]
 
-  def to_pyg_data(self):
+  def to_pyg_data(self, target: np.ndarray):
```

Comment:
Perhaps we should just go ahead and add a test for this. We'll have to add pytorch-geometric to `.travis.yml` but that might be worth the additional work to ensure correctness